azerothcore/azerothcore-wotlk
# feat(Core/Achievements): Send guild's name instead of the player's on…

…e when achieved realm first kill.

Fixed #5369.




## Changes Proposed:
-  Added condition: if player has guild send its name to the server for realm first kill achievements.


## Issues Addressed:
- Closes #5369.



## Tests Performed:

- Tested ingame with two different chars and different achievements.


## How to Test the Changes:

1. Create char.
2. Create your own guild by command: `.guild create "name_of_guild"`.
3. `.achiev add 4576`.
4. Observe chatt.


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
